src/components/component-playground.jsx

```diff
@@ -270,6 +270,10 @@ module.exports = React.createClass({
     window.addEventListener('resize', this.onWindowResize);
 
     this._updateContentFrameOrientation();
+
+    if(this.props.component) {
+      this.refs["componentName-" + this.props.component].getDOMNode().scrollIntoView(); 
```

Comment:
`scrollIntoView` options are only supported on Firefox, sadly. Should i still add it?